checkov/sast/checks/base_registry.py

```diff
@@ -46,8 +52,21 @@ def _load_checks_from_dir(self, directory: str, sast_languages: Set[SastLanguage
                         logging.warning(f'cant parse rule file {file}')
                         continue
                     for rule in rules:
+                        check = Registry._get_check_from_rule(rule)
+                        if not check:
+                            break
+                        if not self.runner_filter.should_run_check(check):
```

Comment:
the `runner_filter` value can be None (you set it in the constructor) so you will have an exception in this case